### DISCUSSION TITLE
Zapret на windows 10 под arm64

### DISCUSSION THREAD

**Discussion by Mike-O-0** [bol-van/zapret#186] [General]
Видел на ntc пост про тестирование под арм на винде. Могу для тестов предоставить Xiaomi Mi Pad 5 (WiFi версия) с разлоченным загрузчиком, на 4pda вроде народ накатывает винду на него без проблем. Территориально - Зеленоград. Ответ можно в почту free4telecom@gmail.com.

**Comment by bol-van** [comment]
Это не настолько важно пока, чтобы ехать в другой город.
В любом случае даже если оно и заработает, то потребует перманентного режима testsigning с надписью на десктопе
Или взлома bootmgr/winload, чтобы убрать запрет на постоянный сеттинг nointegritychecks

**Comment by bol-van** [comment]
Было бы более полезно получить удаленный доступ к win 11 arm 64 виртуалке, запущенной под mac m1/m2, например
Или win11 на малинке
win10, увы, не поддерживает x64 emulation

**Reply by Tsifrokayf** [reply]
11 поддерживает
